app/form_objects/initial_request_form.rb

```diff
@@ -4,7 +4,11 @@ class InitialRequestForm
   attr_accessor :training_provider_code, :training_provider_query
 
   def add_no_results_error
-    errors.add(:training_provider_query, "We couldn't find this provider, please check your input and try again")
+    support_email = "becomingateacher@digital.education.gov.uk"
+
+    errors.add(:training_provider_query,
+               "We couldn't find this organisation - please check your information and try again.
+                To add a new organisation to Publish, contact #{support_email}.")
```

Comment:
Maybe we could do 
```suggestion
                To add a new organisation to Publish, contact #{Settings.service_support.contact_email_address}.")
```
to keep the email in one place

Reply:
does it need mailto?

Reply:
@AgaDufrat, awesome suggestion 💯 

@defong I was thinking about making it a mailto but there are two problems with that.

- The form builder doesn't support html in error messages.
- I don't think we should include markup in an error message because in the error summary the error message itself is a link so by adding a mailto link we would end up with a link inside a link